src/common/tse.c

```diff
@@ -642,12 +644,19 @@ tse_task_post_process(tse_task_t *task)
 		if (task_tmp->dt_result == 0 && !dtp_tmp->dtp_no_propagate)
 			task_tmp->dt_result = task->dt_result;
 
+		dsp_tmp = dtp_tmp->dtp_sched;
+		diff_sched = dsp != dsp_tmp;
+
+		if (diff_sched) {
```

Comment:
probably worth a D_INFO or WARNING here.

Reply:
well, it is formally support the use case of task/dep_task on different schedulers, maybe fine to add a DEBUG log but maybe need not add INFO or WARNING.